### PR TITLE
Product Tab block layout and hover

### LINKED ISSUE #668: [Product]  Tab block layout and hover
On the tab block there are two issues:

- The height, if the Hero block is shorter or longer then the tab doesn't map to it
- On hover, the icons do not change colour

https://main--danaher-ls-aem--hlxsites.hlx.page/us/en/products/sku/548021-molecular-devices

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #668 and #680

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://668-prod-tab-block-layout--danaher-ls-aem--hlxsites.hlx.page/us/en/products/family/allegra-v-15r-benchtop-centrifuges

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
Looks great.  For the two issues, the icon colour and then the scrolling menu links, we can raise a new task for this.

File: blocks/page-tabs/page-tabs.css [hunk 1]
```diff
@@ -47,11 +49,27 @@ svg symbol path {
...
-  @apply w-36 h-10 rounded-md;
+  @apply w-36 h-10 bg-danaherpurple-500 text-white;
 }
 
 .page-tabs-container.fixed li > a {
-  @apply flex-row;
+  @apply flex-row bg-danaherpurple-500 text-white;
+}
+
+.page-tabs-container.fixed li > a:hover {
```
Comment (duynguyen) [inline blocks/page-tabs/page-tabs.css:59]: do you need there? I think the text color and background of the fixed tabs is the same as the original.

Comment (duynguyen) [inline blocks/page-tabs/page-tabs.css:59]: ok I see the hover colors in fixed tabs are different from the original one. then let's remove the group-hover in the original tabs as well and use `:hover` for them here as well

File: blocks/product-hero/product-hero.js [hunk 2]
```diff
@@ -127,6 +130,22 @@ export default async function decorate(block) {
...
+    const infoDiv = div();
+    if (response[0]?.raw.externallink !== undefined) {
+      infoDiv.prepend(
+        p('For additional information'),
+        a(
+          { href: `${response[0]?.raw.externallink}?utm_source=dhls_website`, target: '_blank' },
+          span({ class: 'ext-link' }),
+        ),
+      );
+      infoDiv.querySelector('a .ext-link').innerHTML = `Visit ${response[0]?.raw.opco}<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="inline-block w-5 h-5 pb-1">
```
[the remark this answers is not in the document]
Comment (sharanagoudapatil) [reply blocks/product-hero/product-hero.js]: done.

**Review by duynguyen — APPROVED** [review]

**Pull request merged**